ci_framework/roles/tempest/tasks/main.yml

```diff
@@ -32,6 +32,15 @@
   ansible.builtin.include_tasks: configure-tempest.yml
   when: not cifmw_tempest_dry_run | bool
 
+- name: Copy CA bundle to cifmw_tempest_artifacts_basedir
+  ansible.builtin.copy:
+    src: "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+    dest: "{{ cifmw_tempest_artifacts_basedir }}"
+    mode: '0444'
+    owner: "{{ lookup('env', 'USER') }}"
+    group: "{{ lookup('env', 'USER') }}"
+    remote_src: true
```

Comment:
if we are ok to run the tempest pod as root, we could remove this and use `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem` in the volume list bellow.